### PR TITLE
Bring back Cancel operation button on deployment group detail page

### PR DESCRIPTION
**Pull request by alebellu**
Closes SixSq/tasklist#2617

### CODE REVIEW

File: code/src/cljs/sixsq/nuvla/ui/deployment_sets_detail/subs.cljs [hunk 1]
```diff
@@ -34,7 +34,7 @@
   ::deployment-set
   :<- [::deployment-set-stored-and-edited]
   (fn [[stored edited]]
-    (merge stored edited)))
```
Comment (0xbase12) [inline code/src/cljs/sixsq/nuvla/ui/deployment_sets_detail/subs.cljs:37]: Is this change related to this PR?

Comment (alebellu) [reply code/src/cljs/sixsq/nuvla/ui/deployment_sets_detail/subs.cljs:37]: Yes. After executing an action, e.g Start,the Cancel button was not enabled in the ui, even though the operation was returned from the server. I found that it was because the `operations` attribute in `edited` was overriding the `operations` attribute in stored. So I changed the merge operation to only include from `edited` the attributes that can be edited by the user, and that fixed this issue .
